### DISCUSSION TITLE
How to get related meta-tegra branch

### DISCUSSION THREAD

**Discussion by NavSriv** [OE4T/meta-tegra#1474] [Q&A] [answered]
I am using [Diamond Systems Jetson Orin NX 16GB](https://www.diamondsystems.com/products/jackson) board for development. They are using Kernel version - 5.10.104, BSP version - Jetpack 35.3.1 - R35 (release)

How can I search for related meta-tegra branch or tag?

I used Langadale but it uses 35.2.1 and kirkstone uses 35.4.1. Appreciate suggestions. Thank you

**Comment by madisongh** [comment, marked as answer]
[This wiki page](https://github.com/madisongh/meta-tegra/wiki/Which-branch-should-I-use%3F) has information on the branches we're maintaining.  We try to align with the [Yocto Project's release timelines](https://wiki.yoctoproject.org/wiki/Releases).
